core/src/main/java/io/th0rgal/oraxen/mechanics/provided/gameplay/noteblock/NoteBlockMechanicListener.java

```diff
@@ -16,6 +16,7 @@
 import io.th0rgal.oraxen.utils.breaker.BreakerSystem;
 import io.th0rgal.oraxen.utils.breaker.HardnessModifier;
 import io.th0rgal.protectionlib.ProtectionLib;
+import net.minecraft.world.InteractionResult;
```

Comment:
This introduces an NMS-class to non-nms packages
In theory this should be fine aslong as this class exists in all versions
If there is an API equivalent I would like to see it used instead

Reply:
pretty sure it exists from all the way to 1.18 to 1.20.4